### PR TITLE
Add doc page for pipeline configuration (by TOML file)

### PR DESCRIPTION
**Pull request by tommyblue**
#### :question: What

Add the "Pipeline configuration" doc page to the website

### CODE REVIEW

File: content/en/docs/Core concepts/toml.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+In fact, most pipelines don't want to send the full records to the output, but they will select
+a few important columns out of the many available columns.  
+Notice that this is just a selection: it is up to the output component to decide how to
+physically serialize those columns. For instance, the `DynamoDB` component requires the user
+to specify an option called columns that specifies the name and the type of the column where
+the fields will be written.
+
+#### Metrics configuration
+
+The `[metrics]` section permits to configure the metrics host to use. Currently, only `datadog` is
```
Comment (arl) [inline content/en/docs/Core concepts/toml.md:126]: propositions:
s/permits/allows
s/host/backend

File: content/en/docs/Core concepts/toml.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+[[filter]]
+name="filterB"
+```
+
+`[output]` selects the output component; the output is where records that made it to the end of
+the filter chain without being discarded end up. In this case, the `DynamoDB` output is selected,
+and its configuration is specified in `[output.config]`.
+
+The `fields` option in the `[output]` section selects which fields of the record will be send
```
Comment (arl) [inline content/en/docs/Core concepts/toml.md:115]: tense

**Review by arl — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+setup: ## Setup the website for local development or to add content
+	git submodule add --force https://github.com/google/docsy.git themes/docsy
+	git submodule update --init --recursive
+	npm install postcss-cli autoprefixer postcss
+
+dev: ## Run local server to check you content while writing
+	hugo server
+
+build: ## Build the static files of the website
+	hugo
```
[the remark this answers is not in the document]
Comment (tommyblue) [reply Makefile:15]: ok, I'll add a ticket for that

File: content/en/docs/Core concepts/toml.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+### Pipelines
+
+A pipeline is the configured set of operations that Baker performs during its execution.
+
+It is defined by:
+
+* One input component, defining where to fetch records from
+* Zero or more filters, which are functions that can process records (reading/writing fields,
+clearing them or even splitting them into multiple records)
+* One output component, defining where to send the filtered records to (and which columns)
```
[the remark this answers is not in the document]
Comment (tommyblue) [reply content/en/docs/Core concepts/toml.md:18]: ok, fixed also in other places in the doc

File: content/en/docs/Core concepts/toml.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+* One optional upload component, defining where to send files produced by the output component (if any)
+
+Notice that there are two main usage scenarios for Baker:
+
+* Baker as a batch processor. In this case, Baker will go through all the records that are fed
+by the input component, process them as quickly as possible, and exit.
+* Baker as a daemon. In this case, Baker will never exit; it will keep waiting for incoming
+records from the input component (e.g.: Kinesis), process them and send them to the output.
+
+Selecting between scenario 1 or scenario 2 is just a matter of configuring the pipeline; in fact, it is the input component that drives the scenario. If the input component exits at some point, Baker will flush the pipeline and exit as well; if instead the input component is endless, Baker will never exit and thus behave like a daemon.
```
[the remark this answers is not in the document]
Comment (tommyblue) [reply content/en/docs/Core concepts/toml.md:28]: rephrased a bit the whole paragraph

File: content/en/docs/Core concepts/toml.md (hunk 2 above)
Comment (tommyblue) [reply content/en/docs/Core concepts/toml.md:115]: ok, I also remove all other futures

**Pull request merged**